core/services/ocr2/plugins/ocr2keeper/evm/mercury_lookup.go

```diff
@@ -241,6 +307,7 @@ func (r *EvmRegistry) singleFeedRequest(ctx context.Context, ch chan<- MercuryBy
 				// there are 2 possible causes for 404: incorrect URL and querying a block where report has not been generated
 				r.lggr.Errorf("MercuryLookup upkeep %s block %s received status code %d for feed %s", upkeepId.String(), ml.query.String(), resp.StatusCode, ml.feeds[index])
 				// return 404 for retry
+				retryable = true
```

Comment:
we should also have a block 
```
// if the server is down, it will return 500 and we should retry later
			if resp.StatusCode == http.StatusInternalServerError {
				retryable = true
			}
```